### PR TITLE
Pin cuda-python >=11.7.1,<12

### LINKED ISSUE #16: @conda-forge-admin, please rerender
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #16.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #16

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

**Comment by leofang** [comment]
cc: @kkraus14

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,10 +13,12 @@ source:
   sha256: c8e9d769a6066bb5f94bb0362dbc20ed6f6b525336373263aeb6f9c6dc7b91e4
 
 build:
-  number: 0
+  number: 1
   skip: true  # [py<37 or osx or cuda_compiler_version != "11.7"]
   script:
     - {{ PYTHON }} -m pip install . --no-deps -vv
+  run_exports:
+    - {{ pin_subpackage('cuda-python') }}  # we need this because CUDA Python 11.7.1 has ABI breaks, see conda-forge/cuda-python-feedstock#15
```
[the remark this answers is not in the document]
Comment (leofang) [reply recipe/meta.yaml:21]: @kkraus14 this achieves the same effect. `pin_subpackage` has `min_pin` default to `x.x.x.x.x.x` and `max_pin` default to `x`.

[the remark this answers is not in the document]
Comment (leofang) [reply recipe/meta.yaml:21]: Ah I see what you mean. We don't want >=11.7.2,<12 for cuda-python 11.7.2. Will fix.

[the remark this answers is not in the document]
Comment (leofang) [reply recipe/meta.yaml]: Fixed in [834ee3c](https://github.com/conda-forge/cuda-python-feedstock/pull/17/commits/834ee3c93b23670ac03e478d80a02cb1e6661c9b).

**Review by kkraus14 — APPROVED** [review]

**Pull request merged**